docs/release-notes/deprecate-priority-scheduler.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Deprecations**
+
+Kubernetes Scheduling: Support for the priority scheduler with preemption for Kubernetes is discontinued
+due to limited usage. Users should transition to the default scheduler.
```

Comment:
This looks good. Based on how we announced the agentrm deprecations, I have a couple small suggestions.
- I think we could be a little more clear about the state of the feature for this release: we're announcing that it could be removed in future releases. 
- We've been trying to add links to release notes to make things more discoverable. Is there a master config reference we could point to?

I've also been using "Resource Manager" a lot in the release notes in an attempt to add clarity.

What do you think of this?
```
Support for the priority scheduler with Kubernetes Resource Managers is discontinued and may be removed in a future release. Users should transition to the default scheduler. Visit < link to master config reference > for details. 
```

If we're removing the entire priority scheduler (as opposed to a piece of it), I would just use "priority scheduler" instead of "priority scheduler with preemption".